app/grandchallenge/workstations/admin.py

```diff
@@ -36,6 +42,50 @@ class SessionHistoryAdmin(SimpleHistoryAdmin):
     ]
 
 
+class FeedbackAdmin(ModelAdmin):
+    readonly_fields = (
+        "user_comment",
+        "session",
+        "screenshot",
+        "context",
+        "github_link",
+    )
+    list_display = ("session", "github_link")
+    search_fields = [
+        "session__pk",
+        "session__creator__username",
+        "user_comment",
+    ]
+
+    class Meta:
+        model = Feedback
+
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.select_related("session__creator")
```

Comment:
Nit: there is `list_select_related` for this https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_select_related